### PR TITLE
Update Set-SPOTenantTaxonomyReplicationParameters.md

### PR DESCRIPTION
**Pull request by PowershellScripts**
Suggested text for missing parameter descriptions

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantTaxonomyReplicationParameters.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ Tenant admin can also reset to replicate all the groups.
 
 ### -ReplicateAllGroups
 
-PARAMVALUE: SwitchParameter
+The parameter specifies whether all groups should be replicated.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantTaxonomyReplicationParameters.md:62]:
```suggestion
This parameter specifies whether all groups should be replicated.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantTaxonomyReplicationParameters.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ Accept wildcard characters: False
 
 ### -ReplicatedGroups
 
-PARAMVALUE: String[]
+Specifies a comma separated list of groups that should be replicated.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantTaxonomyReplicationParameters.md:78]:
```suggestion
Specifies a comma-separated list of groups that should be replicated.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @PowershellScripts, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@get-itips @dariomws @tseward

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**